### PR TITLE
Initialize Rails and create data model

### PR DESCRIPTION
**Pull request by mullenlucas**
## Implemented features:

- Initialized a Ruby on Rails web application using PostgreSQL as the database framework
- Created database schema by using migrations according to the ERB diagram (see Readme file)
- Created relationships between the respective data types
- Implemented git workflow guidelines
- Added Linter configuration files for RoR projects
- Updated readme file

### CODE REVIEW

File: db/migrate/20221130183840_add_user_ref_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221130183840_add_user_ref_to_comments.rb]: - [x] according to project requirement kindly change `user` to `author` i.e `t.references :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20221130183914_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221130183914_add_user_ref_to_likes.rb]: - [x] according to project requirement kindly change `user` to `author` i.e `t.references :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20221130184003_add_user_ref_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221130184003_add_user_ref_to_posts.rb]: - [x] according to project requirement kindly change `user` to `author` i.e `t.references :author, foreign_key: { to_table: 'users' }`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @mullenlucas  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @mullenlucas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**